### ISSUE TITLE
Populator docs or examples?

### ISSUE THREAD

**Issue by mgraham** [trailblazer/representable#228]
I'm trying to parse a JSON document into an object graph that is quite different in structure.

I think populators are what I'm looking for, but there isn't any documentation for them.

Does somebody have any code that shows how populators work?

**Comment by apotonick** [comment]
Hi Michael, we're slowly fading out Representable in favor or `Transform`, a new gem that is much simpler. So, your PRs are appreciated, but keep in mind the gem's going to be replaced "somewhere this year"™.

**Comment by mgraham** [comment]
Ok, noted - thanks!

(I'm assuming `Transform` isn't ready enough yet that I should start off with it instead of Representable for a new project...?)

**Comment by apotonick** [comment]
Yes, go with Representable and follow our Facebook page https://www.facebook.com/trailblazer.to - we're announcing new/old infos there!
